test/Math/rlexe.xml

```diff
@@ -42,4 +42,9 @@
       <baseline>clz32.baseline</baseline>
     </default>
   </test>
+  <test>
+    <default>
+      <files>max2.js</files>
```

Comment:
To test both the Js version and the CPP fallback should put the test into this file twice, once as you've put it AND a repeat with `<compile-flags>-JsBuiltin-</compile-flags>`